rest_device.go

```diff
@@ -307,11 +315,12 @@ func mapDeviceClusterDetailsDomainToAPI(clusterDetails *ClusterDetails) *api.Clu
 	}
 	req := api.ClusterDetailsRequest{}
 	req.ClusterName = clusterDetails.ClusterName
-	req.NumOfNodes = clusterDetails.NumOfNodes
 	clusterNodeDetailsRequest := make(map[string]api.ClusterNodeDetailRequest)
 	for k, v := range clusterDetails.ClusterNodeDetails {
```

Comment:
This for loop is no longer required since clusterNodeDetailsRequest["node0"] and clusterNodeDetailsRequest["node1"] is being defined below

Reply:
Fixed